klee/include/klee/ExecutionState.h

```diff
@@ -78,6 +78,9 @@ class ExecutionState : public IAddressSpaceNotification {
     /// Disables forking, set by user code.
     bool forkDisabled;
 
+    /// Whether to add the constraints at branches when forking is disabled
+    bool addConstraintWhenForkingIsDisabled;
```

Comment:
Or you can mark this as true by default, then we can merge it as it is enabled by default.

Reply:
I've made it an argument that defaults to true.